akka-stream/src/main/scala/akka/stream/impl/PhasedFusingActorMaterializer.scala

```diff
@@ -600,6 +618,24 @@ private final case class SavedIslandData(
   override def makeLogger(logSource: Class[_]): LoggingAdapter =
     Logging(system, logSource)
 
+  /**
+   * INTERNAL API
+   */
+  @silent("deprecated")
+  @InternalApi private[akka] override def actorOf(context: MaterializationContext, props: Props): ActorRef = {
+    val effectiveProps = props.dispatcher match {
+      case Dispatchers.DefaultDispatcherId =>
+        props.withDispatcher(context.effectiveAttributes.mandatoryAttribute[ActorAttributes.Dispatcher].dispatcher)
+      case ActorAttributes.IODispatcher.dispatcher =>
+        // this one is actually not a dispatcher but a relative config key pointing containing the actual dispatcher name
+        // FIXME go via attributes here,or something
```

Comment:
Where does this method come from and how will we fix the FIXME? (follow up ticket?)

Reply:
It is fixed by the indirection introduced for dispatchers earlier - when trying to load a dispatcher and that config key is a string the dispatcher lookup will just treat that string as a new config key to look up instead.

We have test coverage for this in `AttributeSpec`, I just forgot to remove this comment.

Reply:
Dang, I didn't read the code carefully enough, it doesn't go through attributes, I'll fix it.

Reply:
Great, even better ;)